config/plugin/network_problem.sh

```diff
@@ -1,23 +1,35 @@
 #!/bin/bash
 
 # This plugin checks for common network issues. Currently, it only checks
-# if the conntrack table is full. 
+# if the conntrack table is full.
 
 OK=0
 NONOK=1
 UNKNOWN=2
 
-[ -f /proc/sys/net/ipv4/netfilter/ip_conntrack_max ] || exit $UNKNOWN
-[ -f /proc/sys/net/ipv4/netfilter/ip_conntrack_count ] || exit $UNKNOWN
+# "nf_conntrack" replaces "ip_conntrack" - support both
+NF_CONNTRACK_COUNT='/proc/sys/net/netfilter/nf_conntrack_count'
```

Comment:
nit: Could you add `readonly ` before the [constants](https://google.github.io/styleguide/shell.xml#Constants_and_Environment_Variable_Names) (`NF_CONNTRACK_COUNT`, `NF_CONNTRACK_MAX`...) and the [read-only variables](https://google.github.io/styleguide/shell.xml#Read-only_Variables) (`conntrack_count`, `conntrack_max`)?

Thank you!

Reply:
I added `readonly` where necessary. 
Feels like a lot of `readonly` for a small script (readability wise) but I understand the reasoning :)